README.md

```diff
@@ -119,6 +119,8 @@ protoc-gen-connect-openapi also has support for the [OpenAPI v3 annotations](htt
 | proto | - | Generate requests/repsonses with the protobuf content type |
 | services | - | Filter which services have OpenAPI spec generated. The default is all services. Comma-separated, uses the full path of the service "[package name].[service name]" |
 | trim-unused-types | - | Remove types that aren't references from any method request or response. |
+| use-short-service-name-for-tags | - | Use the short service name (Name()) instead of the full name (FullName()) for OpenAPI tags. |
+| use-short-names-for-operation-id | - | Set the operationId to shortServiceName + "_" + method short name instead of the full method name. |
```

Comment:
Can we make these slightly shorter? Maybe `short-service-tags` and `short-operation-ids`?

Reply:
Yeah good point, I've shortened the options.